notebooks/official/bigquery-ml/bqml-vertexai-model-registry.ipynb

```diff
@@ -0,0 +1,1465 @@
+{
```

Comment:
<h2>model to an endpoint -&gt;</h2><h2><br></h2><h2>Vertex AI Model resource to a Vertex AI Endpoint resource</h2>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/842/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>